conan/api/subapi/profiles.py

```diff
@@ -106,6 +105,10 @@ def _get_profile(self, profiles, settings, options, conf, cwd, cache_settings,
         profile.conf.rebase_conf_definition(global_conf)
         return profile
 
+    def load_profile(self, profiles, settings, options, conf, cwd):
```

Comment:
This could be a bit confusing, we already have the ``get_profile()`` with same arguments. It seems the difference is the application of profile_plugin, isn't it? I think we want to discuss this (future) public API